ray_provider/hooks/ray.py

```diff
@@ -56,7 +58,7 @@ def get_connection_form_widgets(cls) -> dict[str, Any]:
 
         return {
             "address": StringField(lazy_gettext("Ray dashboard url"), widget=BS3TextFieldWidget()),
-            "create_cluster_if_needed": BooleanField(lazy_gettext("Create cluster if needed")),
+            # "create_cluster_if_needed": BooleanField(lazy_gettext("Create cluster if needed")),
```

Comment:
Why is it commented out?

Reply:
Its an input parameter for Ray JobSubmission SDK that requires an external job scheduler. We will not need it.